lib/tests/modules/error-mkOption-in-submodule-config.nix

```diff
@@ -0,0 +1,12 @@
+{ lib, ... }:
+let
+  inherit (lib) mkOption;
+in
+{
+  options.sub = lib.mkOption {
+    type = lib.types.submodule {
+      wrong2 = mkOption {};
```

Comment:
I'm so happy that past me introduced this error message because when I first looked at the code here I wondered what was wrong with this expression ;-)

Reply:
Yeah it doesn't look bad!